index.html

```diff
@@ -110,6 +112,14 @@ <h2>Who are we?</h2>
                 </div>
             </div>
         </div>
+        <div class="sj2-wrapper width-wrapper">
+            <div class="sj2-logo-wrapper">
+                <img class="sj2-logo icon">
+                <img class="sj2-logo text" src="images/sj2/SJ_Text-Logo.png">
+            </div>
+            <div class="sj2-info">Our upcoming Splatoon 2 major!</div>
+            <div class="sj2-reg-button" onclick="window.open('https://www.start.gg/sj2', '_blank');">Learn more & register</div>
```

Comment:
Middle clicking on this button to open the link in a new tab does not work, and there's no way to copy the link URL. I'd suggest using an <a> tag here.